thrift/src/test/java/com/linecorp/armeria/server/thrift/ThriftServiceTest.java

```diff
@@ -130,8 +130,10 @@ void testSync_HelloService_hello(SerializationFormat defaultSerializationFormat)
         client.send_hello(FOO);
         assertThat(out.length()).isGreaterThan(0);
 
-        final THttpService service = THttpService.of(
-                (HelloService.Iface) name -> "Hello, " + name + '!', defaultSerializationFormat);
+        final THttpService service = THttpService.builder()
+                    .addService((HelloService.Iface) name -> "Hello, " + name + '!')
+                    .defaultSerializationFormat(defaultSerializationFormat)
+                    .build();
```

Comment:
Would you mind reverting the changes where you used the builder API even if it's doable with the undeprecated static factory methods, so that the static factory methods are tested properly?